src/instruments/src/Common/hooks.tsx

```diff
@@ -57,7 +57,7 @@ export const useInteractionEvents = (events: string[], handler: (any?) => void):
         return () => {
             events.forEach((event) => getRootElement().removeEventListener(event, wrappedHandler));
         };
-    }, [
-        ...events,
+    }, [ /* eslint-disable-line react-hooks/exhaustive-deps */
+        ...events, /* eslint-disable-line react-hooks/exhaustive-deps */
     ]);
```

Comment:
This is caused by ESLint not being able to verify the dependency array, since we are using a spread operator.

![image](https://user-images.githubusercontent.com/4503241/141664750-986a4b0b-757a-43a3-bae7-e92354b96ec8.png)

I can't really think of a good way to fix this, other than create a custom hook to memoize the dependencies that would have an ESLint exception inside it to circumvent the warning in the call site.

Reply:
To be specific I really meant the double comment. The spread operator seems acceptable to me.

Reply:
Fixed